.pa11yci.json

```diff
@@ -0,0 +1,4 @@
+{
+  "hideElements": ".typography .inner *, #comments",
```

Comment:
Let's see if we can use `.post .typography .inner` instead of `.typography .inner *`.